app/helpers/claims_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ClaimsHelper
+  def peer_to_peer_preferred_contact_details
+    preferred_contact_method = peer_to_peer_preferred_contact_method
+    return if preferred_contact_method.blank?
+
+    if preferred_contact_method.name == "Email"
+      current_user.person&.email
+    else
+      current_user.person&.phone
+    end
+  end
+
+  def peer_to_peer_preferred_contact_method
+    @peer_to_peer_preferred_contact_method ||= current_user.person&.preferred_contact_method
+  end
+end
```

Comment:
Some of this logic already exists in `Person.preferred_contact_info`. Moreover, i think it could be more straightforward, easier to follow, if we explicitly set locals in the controller action instead of relying on a helper to calculate them from the view.

I spiked out what this could look like in https://github.com/rubyforgood/mutual-aid/commit/b458ebe09886dfebf34920954a12d8d27891ba13. LMK what you think and feel free to cherry-pick or merge that commit if it works.

Reply:
@exbinary Thanks for that commit, I have cherry-picked it and made a minor change which uses safe nav for picking preferred_contact_method_id https://github.com/rubyforgood/mutual-aid/commit/b458ebe09886dfebf34920954a12d8d27891ba13#diff-30e7c2adf26639d802013e232d411680R10

Reply:
🙌🏾